### PR TITLE
Added UNDERWATER_DISTORTION toggle to new quality setting file

### PR DESCRIPTION
**Pull request by OpenSauce04**
### Summary:
As the title says, I've added a preset toggle that removes the underwater distorting
This PR also uses this new preset option to disable underwater effects on the "lowest" subpack option

### CODE REVIEW

**Comment by OpenSauce04** [comment]
This PR is now ready for review

**Comment by OpenSauce04** [comment]
Don't pull yet, I'm going to seperate the waving effect from the caustics before this is pulled

**Comment by jebbyk** [comment]
ok

**Comment by OpenSauce04** [comment]
@jebbyk This is ready to be checked and pulled

**Review by jebbyk — CHANGES_REQUESTED** [review]
It is better to place actual distortion calculation in to #if ..... #endif because in this case if UNDERWATER_DISTORTION is not defined  this code will not be even compiled and will not affect performance. Currently even if distortion is disabled this code continues to calculate "distortion" which is equals 0.0.

**Comment by OpenSauce04** [comment]
@jebbyk I've tested it and it works absolutely fine, but I can change it

**Comment by jebbyk** [comment]
@OpenSauce04 Ye Im sure it works, but I'm talking about performance.

**Comment by OpenSauce04** [comment]
Ok, I'll work on it now

**Review by jebbyk — APPROVED** [review]

**Comment by jebbyk** [comment]
I'll check this on my device and merge when have time :-)

**Comment by jebbyk** [comment]
@OpenSauce04 Hi, can you resolve conflicts?

**Comment by OpenSauce04** [comment]
Oh there are conflicts?
Must be due to that other PR, it was fine beforehand
Will fix them

File: subpacks/lowest/shaders/glsl/includes/options/quality_settings/effects/underwater_distortion.glsl [hunk 1]
```diff
@@ -0,0 +1 @@
+//#define UNDERWATER_DISTORTION //Comment this line to disable underwater distortion
```
Comment (jebbyk) [inline subpacks/lowest/shaders/glsl/includes/options/quality_settings/effects/underwater_distortion.glsl:1]: I dont think this option is heavy enough to be placed in quality settings

Comment (OpenSauce04) [reply subpacks/lowest/shaders/glsl/includes/options/quality_settings/effects/underwater_distortion.glsl:1]: What does that even mean?

Comment (OpenSauce04) [reply subpacks/lowest/shaders/glsl/includes/options/quality_settings/effects/underwater_distortion.glsl:1]: > I dont think this option is heavy enough to be placed in quality settings

What do you mean by that?

Comment (jebbyk) [reply subpacks/lowest/shaders/glsl/includes/options/quality_settings/effects/underwater_distortion.glsl:1]: @OpenSauce04 quality change should affect performance, but this option I think will not affect performance at all. But maybe I'm mistaking

Comment (OpenSauce04) [reply subpacks/lowest/shaders/glsl/includes/options/quality_settings/effects/underwater_distortion.glsl:1]: It definitely seems like something that would fit into the quality settings, however of you feel strongly enough that it should be moved to preset options, we could do that
I was under the impression that it did affect performance though

Comment (jebbyk) [reply subpacks/lowest/shaders/glsl/includes/options/quality_settings/effects/underwater_distortion.glsl:1]: ok, anyway I'm currently preparing this PR to merge into dev

**Pull request merged**